src/main/java/Decide.java

```diff
@@ -8,6 +8,8 @@
  */
 public class Decide {
 
+    private boolean[] CMV = new boolean[15];
```

Comment:
This dosen't feel like it belongs to this PR?

Reply:
See slack, we thought it would be easier to have it here

Reply:
Okay 👍